content/docs/user-guide/data-management/remote-storage/hdfs.md

```diff
@@ -47,6 +47,14 @@ them with the `--local` option, so they're written to a Git-ignored config file.
                               kerb_ticket /path/to/ticket/cache
   ```
 
+- `replication` - replication factor for write operations on HDFS cluster.
+  default value is 3.
```

Comment:
one more minor comment - it should be Default (since it comes after period).

Reply:
Fixed.